phabricatoremails/source.py

```diff
@@ -0,0 +1,73 @@
+import json
+from dataclasses import dataclass
+from typing import Optional
+
+import requests
+from requests import RequestException
+
+
+class PhabricatorException(Exception):
+    pass
+
+
+@dataclass
+class PhabricatorSource:
+    server: str
+    token: str
+    override_limit: Optional[int]
+
+    def _get(self, endpoint, **query_parameters):
+        query_parameters = {k: v for k, v in query_parameters.items() if v}
+        url = "{}/api/{}".format(self.server, endpoint)
+
+        body = {
+            "params": json.dumps(
+                {"__conduit__": {"token": self.token}, **query_parameters}
+            )
+        }
+
+        try:
+            result = requests.post(url, data=body)
```

Comment:
There is a `post` call inside a method named `_get`. If this is intentional (i.e. this is something hacky that we can not get around), there should be clear documentation about why this is the case.

Reply:
Good eye :eyes:, changed this from a `requests.get(...)` earlier and forgot to update the method